modules/core/src/effects/post-process-effect.ts

```diff
@@ -1,19 +1,17 @@
-import type {Device} from '@luma.gl/core';
-import {normalizeShaderModule} from '@luma.gl/shadertools';
-import type {Framebuffer} from '@luma.gl/core';
+import type {Device, Framebuffer} from '@luma.gl/core';
+import {normalizeShaderModule, ShaderPass} from '@luma.gl/shadertools';
 
 import ScreenPass from '../passes/screen-pass';
 
 import type {Effect, PostRenderOptions} from '../lib/effect';
-import type {ShaderModule} from '../types/types';
 
 export default class PostProcessEffect implements Effect {
   id: string;
   props: any;
-  module: ShaderModule;
+  module: ShaderPass;
```

Comment:
I will add a default type param for ShaderPass which will get rid of this error.